### ISSUE TITLE
Output fewer newlines when validation is successful

### ISSUE THREAD

**Issue by zeke** [Redocly/openapi-cli#48]
⚠️ Very minor issue alert! ⚠️ 

The output below has a lot of extra newlines. I can see why those newlines would be helpful in cases where there are many errors per schema, but when there are no errors it looks a bit spacious:

```
> openapi validate openapi/*.yml



openapi/api.github.com.yml results. Errors: 0, warnings: 0
    

openapi/ghe-2.15.yml results. Errors: 0, warnings: 0


openapi/ghe-2.16.yml results. Errors: 0, warnings: 0


openapi/ghe-2.17.yml results. Errors: 0, warnings: 0


openapi/ghe-2.18.yml results. Errors: 0, warnings: 0
Total results. Errors: 0, warnings: 0
```

**Comment by knidarkness** [comment]
Thanks for feedback! This was resolved by #51, so I would close this issue.

**Comment by zeke** [comment]
Nice!
